test/test_less_pycharm.py

```diff
@@ -0,0 +1,448 @@
+import sys
+
+import cheetah
+import numpy as np
+import torch
+
+sys.path.append(
```

Comment:
@FelixTheilen This line is still giving me a bit of a headache. Why exactly is it there?

Once that's fixed, I think I can merge this pull requests.